packages/stream/index.js

```diff
@@ -175,6 +177,7 @@ export const prime = (generator, primer) => {
     },
     /** @param {Error} error */
     async throw(error) {
+      await null;
```

Comment:
For the changes in this file, I would prefer to disable the lint rule and keep this as-is. I don’t see a practical problem with the current formulation and do see an extra turn of the even loop in what might be a hot path.

Reply:
Done